readme.txt

```diff
@@ -5,6 +5,7 @@
 = [4.9.4] TBD =
 
 * Tweak - Keep track of whether the current request was authorized by the Promoter connector [117668]
+* Tweak - Adjust `determine_current_user` priority used to identify user on REST API [124302]
```

Comment:
Needs to specify that this is for Promoter

Reply:
Changes has been applied, thanks.